sql/handler.cc

```diff
@@ -2658,9 +2660,12 @@ static my_bool xarecover_handlerton(THD *unused, plugin_ref plugin,
         */
         if (info->mem_root)
         {
-          // remember "full" xid too when it's not in mysql format
+          // remember "full" xid too when it's not in mysql format.
+          // MDEV-27342 - Also record the server_id of the transaction. It will be used for
```

Comment:
Please don't refer to MDEV:s. It just litters. If needed they can be always found with git annotate.

Reply:
Updated in commit.